### PR TITLE
Agile methods in open source add on

### PR DESCRIPTION
**Pull request by KensayS**

### CODE REVIEW

File: main.typ [hunk 1]
```diff
@@ -95,8 +95,13 @@ Additionally, Agile projects are designed to deliver working software in short i
...
+As Agile Methodologies integrate into mainstream development due to the positive effects on code quality and quick response time, aspects of agile are emerging into open source development. A trend in open source projects appear to be the adoption of Testing and Continuous Integration. The implementation of automated tests establishes trust among the developers involved with the open source project because passing tests demonstrates that their code works, and that the code does not have to be rewritten by other developers. @Turnu_2006 
+
+In addition to Testing and CI, other Agile practices are also making their way into open source development. Iterative development models are becoming more common, allowing features to be developed, tested, and released in small increments. This approach aligns with the Agile principle of responding quickly to changes and adapting to user feedback. Moreover, collaborative coding and pair programming, inspired by Agile's emphasis on teamwork, are being embraced to improve code quality and foster knowledge sharing within the open source community. This not only helps in maintaining code quality but also in mentoring new contributors​​. @Hron_2022
```
Comment (dyc3) [inline main.typ:101]: > Moreover, collaborative coding and pair programming,

We mention that open source development is mostly asynchronous, so stuff like pair programming coming to open source doesn't really make sense.

**Review by dyc3 — CHANGES_REQUESTED** [review]

File: main.typ [hunk 2]
```diff
@@ -95,7 +95,13 @@ Additionally, Agile projects are designed to deliver working software in short i
...
+
+User-centric design is another Agile practice that open source projects are increasingly focusing on. By prioritizing user feedback and requirements, these projects ensure that their software meets the needs and expectations of end-users. Furthermore, Agile's flexible and adaptive planning approach is particularly beneficial for open source projects, which often need to pivot based on contributor availability, community interest, and technological shifts​​. @Muhammad_2021
```
Comment (smcdonou02) [inline main.typ:103]: I like this point I would keep this one and make a new pr with the current merge

File: main.typ [hunk 3]
```diff
@@ -95,7 +95,13 @@ Additionally, Agile projects are designed to deliver working software in short i
...
+
+However, applying Agile methodologies in open source contexts is not without its challenges. Coordinating distributed teams, managing contributions from developers with varying expertise, and ensuring consistent communication are significant hurdles. To overcome these challenges, open source projects often tailor Agile practices to their specific environments, such as using asynchronous communication tools and flexible scheduling to accommodate contributors across different time zones. Establishing clear guidelines and documentation also plays a crucial role in effectively onboarding new contributors and maintaining project continuity​​​​.
```
Comment (smcdonou02) [inline main.typ:105]: Love this, good conclusion to the section, also add this to the pr

**Review by smcdonou02 — CHANGES_REQUESTED** [review]
Add last two paragraphs to where main currently is adding to my section

**Review by dyc3 — APPROVED** [review]

**Review by smcdonou02 — APPROVED** [review]

**Pull request merged**